spring-cloud-gcp-samples/spring-cloud-gcp-data-spanner-sample/README.adoc

```diff
@@ -14,14 +14,14 @@ link:../../spring-cloud-gcp-starters/spring-cloud-gcp-starter-data-spanner[Sprin
 You can define the table via the following DDL
 ```
 CREATE TABLE trades (
+	sequence INT64,
```

Comment:
can we just call it `id`? the `sequence` reminds me of using Oracle DB and also alludes to sequence which is not recommended as it can cause hotspots: https://cloud.google.com/spanner/docs/schema-design#choosing_a_primary_key